pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/tracker/AnalyticsMediaItemTracker.kt

```diff
@@ -6,150 +6,94 @@ package ch.srgssr.pillarbox.player.tracker
 
 import androidx.media3.common.MediaItem
 import androidx.media3.common.Player
-import androidx.media3.common.Player.PositionInfo
+import androidx.media3.common.Tracks
 import androidx.media3.exoplayer.ExoPlayer
 import androidx.media3.exoplayer.analytics.AnalyticsListener
-import ch.srgssr.pillarbox.player.asset.PillarboxData
-import ch.srgssr.pillarbox.player.extension.getPillarboxDataOrNull
-import ch.srgssr.pillarbox.player.tracker.MediaItemTracker.StopReason
+import ch.srgssr.pillarbox.player.PillarboxExoPlayer
+import ch.srgssr.pillarbox.player.extension.getMediaItemTrackerDataOrNull
 import ch.srgssr.pillarbox.player.utils.DebugLogger
-import ch.srgssr.pillarbox.player.utils.StringUtil
-import kotlin.time.Duration.Companion.milliseconds
 
 /**
- * Custom [CurrentMediaItemPillarboxDataTracker.Callback] to manage analytics.
+ * Tracks [Player.getCurrentTracks] to handle [MediaItemTrackerData] changes.
+ * When player is stopped player state = IDLE the MediaPeriod is destroyed and then prepared is called, it will create a session by calling start.
```

Comment:
```suggestion
 * When the player is stopped (player state is `IDLE`), the `MediaPeriod` is destroyed and then `prepare()` is called, it will create a session by calling `start()`.
```